Scripts/parameters/income.py

```diff
@@ -1,24 +1,39 @@
 log_income = {
-    "constant": 6.52186,
+    "constant": 6.8611,
     "zone": {
-        "perc_detached_houses_sqrt": 0.16309,
-        "hw_t": 0.07543,
-        "cbd": 0.13188,
+        "perc_detached_houses_sqrt": 0.18717,
+        "hw": 0.04091,
     },
-    "car_users": 0.21289,
-    "male": 0.20344,
+    "car_users": 0.17134,
+    "male": 0.19845,
     "age_dummies": {
-        "age_30-49": 0.36845,
-        "age_50-64": 0.31557,
+        "age_18-29": 0.0423,
+        "age_30-49": 0.3537,
+        "age_50-64": 0.29101,
     },
     "municipality_dummies": {
-        "Kauniainen": 0.32818,
-        "Vantaa": -0.1037,
-        "Tuusula": -0.13583,
-        "Kerava": -0.07909,
-        "Jarvenpaa": -0.1652,
-        "Sipoo": 0.12297,
-        "Hyvinkaa": -0.13207,
+        "Espoo": 0.07995,
+        "Kauniainen": 0.4515,
+        "Hyvinkaa": -0.08642,
+        "Jarvenpaa": -0.13453,
+        "Sipoo": 0.12064,
+        "Tuusula": -0.09369,
+        "Vihti": -0.07904,
     },
+    "standard_deviation": 0.35499,
+}
+log_income_helsinki = {
+    "constant": 0,
```

Comment:
Is this correct? I think this causes large difference between Helsinki / other.

Reply:
Great that you found this error! The constant had indeed disappeared from Lauri's excel file.